pyproject.toml

```diff
@@ -121,12 +121,8 @@ lint.ignore = [
     "ANN002", # TODO: missing-type-args
     "ANN003", # TODO: missing-type-kwargs
     "ANN201", # TODO: missing-return-type-undocumented-public-function
```

Comment:
Hopefully these will be fixed in the mypy PR

Reply:
Although, more accurately, the static types PR #308